com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDAdditionalCameraData.cs

```diff
@@ -121,11 +131,30 @@ public void CopyTo(HDPhysicalCamera c)
             c.iso = iso;
             c.shutterSpeed = shutterSpeed;
             c.aperture = aperture;
+            c.focusDistance = focusDistance;
             c.bladeCount = bladeCount;
             c.curvature = curvature;
             c.barrelClipping = barrelClipping;
             c.anamorphism = anamorphism;
         }
```

Comment:
As it is a struct now and as we copy all field, this method is obsolete.
Basically if assign the struct to another one, it will copy everything inside.

Reply:
Good point. However this is a public API (and a "blame" shows it has been arround for ~2 years), so I guess it makes sense to keep it for backwards compatibility, right?

Reply:
Right. But perhaps we can start a deprecation plan like putting it as obsolete (warning, explaining why) then later putting it as obsolete (error) then later removing it. (Can be done later)

@sebastienlagarde do we have any file to keep track of deprecated functionality? And do we have any official deprecation plan? Should we invest little time on having a clear deprecation plan in packages in the next roadmap?

Reply:
Ok, I marked the method as obsolete, so no need to keep track of this. When we say we will do somethign later, we never do it in the end...